v2/src/runner.py

```diff
@@ -64,11 +75,12 @@ def _create_commands(spec: _RetrySpec) -> Sequence[str]:
             python_executable=spec.python_executable,
             robot_target=spec.robot_target,
             outputdir=spec.outputdir(),
-            output=str(
-                i
-            ),  # Ensure the `robot` command does not overwrite previous runs
+            # Ensure the `robot` command does not overwrite previous runs
+            output=spec.outputdir() / f"{i}.xml",
```

Comment:
Since we have to pass an absolute path for `previous_output` (robot framework does _not_ prepend `outputdir` if `previous_output` is relative), it seemed to me that it would make sense to use an absolute path here as well, st. these two are for sure in sync.

Reply:
That makes sense... Although I don't like specifying duplicate information inside the struct. Maybe having a helper function, which exposes the output as an absolute path instead?

Reply:
Done.